src/app/(layout-with-navigation)/blog/_components/ImpactCO2ScriptAdder.tsx

```diff
@@ -0,0 +1,26 @@
+'use client'
+
+import Script from 'next/script'
+import { useEffect } from 'react'
+
+function ImpactCO2ScriptAdder() {
+  useEffect(() => {
+    // @ts-expect-error: Loaded by script
+    if (window.impactCO2Detection) {
```

Comment:
Pourquoi on exécute 2 fois cette fonction ?

Reply:
It's a hack but it works man.. Sans ça le script ne fonctionne pas quand on navigue entre les articles du blog

Reply:
Ah oui ça semble logique